File: Gabaritos/N1_2_N2/C/calculo_rapido.c
#include <stdio.h>

int main(void) {
  int s, a, b;
  int qtdNumeros = 0;
  scanf("%d\n%d\n%d", &s, &a, &b);

  for(int i=a; i<=b; i++){
    int soma =0;
    int numero = i;
    while(numero>=10){
      soma += numero%10;
      numero = numero/10;
    }
    soma +=numero;
    if(soma==s) qtdNumeros++;  
  }

  printf("%d" , qtdNumeros);
  
  return 0;
}

File: Gabaritos/Jr/calculo_rapido.py
s = int(input())
a = int(input())
b = int(input())
qtdNumeros = 0

for i in range(a,b+1):
  numero = i
  somaDigitos = 0
  while (numero>=10):
    somaDigitos += numero % 10
    numero = numero // 10
  somaDigitos += numero
  if(somaDigitos==s):
    qtdNumeros += 1

print(qtdNumeros)